docs/proposals/20191017-kubeadm-based-control-plane.md

```diff
@@ -336,7 +336,12 @@ spec:
   - Upgrading machines
 - Scale up operations are blocked based on Etcd and control plane health checks.
   - See [Health checks](#Health checks) below.
-- Scale down operations removes the oldest machine in the failure domain that has the most control-plane machines on it 
+- Scale down operations removes the oldest machine in the failure domain that has the most control-plane machines on it
+- Allow scaling down of KCP with the possibility of marking specific control plane machine(s) to be deleted with annotation key.
+In order to delete a specific machine, user have to annotate a machine(s) that needs to be deleted and KCP implements the
+following:
+  - Filters out a machine(s) with machine delete annotation key.
+  - Makes sure to delete the oldest machine(s) first when more than one machine is marked for deletion.
```

Comment:
This covers the simplistic case related to scale down, depending on the prioritization question I posed above it would also be good to include information how a rollout would be affected by the presence of the annotation.